book/src/chap18.asciidoc

```diff
@@ -27,590 +27,590 @@ For example, this table shows the suits and the corresponding integer codes:
 Test Passed
-julia> @test Card(1, 3) < Card(1, 4)
+julia> @test Carta(1, 3) < Carta(1, 4)
 Test Passed
 ----
 
-+@test+ returns a +"Test Passed"+ if the expression following it is +true+, a +"Test Failed"+ if it is +false+, and an +"Error Result"+ if it could not be evaluated.
++@test+ retorna +"Test Passed"+ se a expressão seguinte for +true+, +"Test Failed"+ se for +false+, e +"Error Result"+ se não puder ser avaliado.
 
 
-=== Decks
+=== Baralhos
 
-Now that we have Cards, the next step is to define Decks. Since a deck is made up of cards, it is natural for each Deck to contain an array of cards as an attribute.
+Agora que temos Cartas, o próximo passo é a definição do Baralhos. Como um baralho é composto de cartas, é natural que cada Baralho contenha um array de cartas como um atributo.
 
-The following is a struct definition for +Deck+. The constructor creates the fields cards and generates the standard set of fifty-two cards:
-(((Deck)))((("type", "programmer-defined", "Deck", see="Deck")))
+A seguir, define-se uma struct para +Baralho+. O construtor cria os campos das cartas e gera o conjunto padrão das cinquenta e dois cartas:
+(((Deck)))((("tipo", "definido pelo programador", "Deck", see="Deck")))
 
 [source,@julia-setup chap18]
 ----
-struct Deck
-    cards :: Array{Card, 1}
+struct Baralho
+    cartas :: Array{Carta, 1}
 end
 
-function Deck()
-    deck = Deck(Card[])
-    for suit in 1:4
-        for rank in 1:13
-            push!(deck.cards, Card(suit, rank))
+function Baralho()
+    baralho = Baralho(Carta[])
+    for naipe in 1:4
+        for valor in 1:13
+            push!(baralho.cartas, Carta(naipe, valor))
         end
     end
-    deck
+    baralho
 end
 ----
 
-The easiest way to populate the deck is with a nested loop. The outer loop enumerates the suits from 1 to 4. The inner loop enumerates the ranks from 1 to 13. Each iteration creates a new +Card+ with the current suit and rank, and pushes it to +deck.cards+.
+A maneira mais fácil de preencher o baralho é com um laço aninhado. O laço externo enumera os naipes de 1 a 4. O laço interno enumera os valores de 1 a 13. Cada iteração cria uma nova +Carta+ com o naipe e o valor correntes e envia-a para +baralho.cartas+.
 
-Here is a +show+ method for +Deck+:
+Aqui está um método +show+ para +Baralho+:
 
 [source,@julia-setup chap18]
 ----
-function Base.show(io::IO, deck::Deck)
-    for card in deck.cards
-        print(io, card, " ")
+function Base.show(io::IO, baralho::Baralho)
+    for carta in baralho.cartas
+        print(io, carta, " ")
     end
     println()
 end
 ----
 
-Here’s what the result looks like:
+Veja como ficou o resultado:
 
 [source,@julia-repl-test chap18]
 ----
-julia> Deck()
+julia> Baralho()
 A♣ 2♣ 3♣ 4♣ 5♣ 6♣ 7♣ 8♣ 9♣ 10♣ J♣ Q♣ K♣ A♦ 2♦ 3♦ 4♦ 5♦ 6♦ 7♦ 8♦ 9♦ 10♦ J♦ Q♦ K♦ A♥ 2♥ 3♥ 4♥ 5♥ 6♥ 7♥ 8♥ 9♥ 10♥ J♥ Q♥ K♥ A♠ 2♠ 3♠ 4♠ 5♠ 6♠ 7♠ 8♠ 9♠ 10♠ J♠ Q♠ K♠
 ----
 
 
-=== Add, Remove, Shuffle and Sort
+=== Adicionar, Remover, Baralhar e Ordenar
```

Comment:
Ambas estão corretas. Se a intenção for ficar um pouco mais "formal", creio que Embaralhar fique melhor.
```suggestion
=== Adicionar, Remover, Embaralhar e Ordenar
```

https://www.comoescreve.com/2019/01/baralhar-ou-embaralhar-qual-voce-usa.html#:~:text=Baralhar%3A%20verbo%20transitivo%20direto%20que,significado%20de%20tornar%20confuso%2F%20confundir.